simplified-viewer-audiobook/src/main/java/org/nypl/simplified/viewer/audiobook/AudioBookPlayerActivity.kt

```diff
@@ -151,6 +151,15 @@ class AudioBookPlayerActivity :
     this.setContentView(R.layout.audio_book_player_base)
     this.playerScheduledExecutor = Executors.newSingleThreadScheduledExecutor()
 
+    /*
+    * Create a new downloader that is solely used to fetch audio book manifests.
+    */
+
+    this.downloadExecutor =
+      MoreExecutors.listeningDecorator(
+        NamedThreadPools.namedThreadPool(1, "audiobook-player", 19)
+      )
+
```

Comment:
@qnga this appears to just be moved up earlier in the activity. Is the problem that it's initialized too late?

Reply:
The real problem is probably the `return` on line 206. The move of the property initialisation should act as a quick fix.